tests/conftest.py

```diff
@@ -0,0 +1,85 @@
+import os
+from unittest.mock import mock_open, patch
+
+import pytest
+
+
+@pytest.fixture()
+def mock_komodo_env_vars():
+    """
+    Provide the environment vars from a 'vanilla' komodo environment.
```

Comment:
Why vanilla with '' here and below? Same with 'fake'...

Reply:
Again, tidied up with more explicit language.

Reply:
Your explanation here is really clear and indeed, this difference is not evident. 
I think they could be arranged in the docstrings of `mock_komodo_env_vars` and `mock_komodoenv_env_vars`, for example, and that would probably be useful for a future developer.

Reply:
Added some clarification to `conftest.py`.